src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -3858,6 +3869,26 @@ namespace SurfaceGeometry {
                         ErrorsFound = true;
                     }
 
+                } else if (Util::SameString(s_ipsc->cAlphaArgs(ArgPointer), "Space")) {
+                    // This is the code for an unmatched "other surface"
+                    // will be set up later.
+                    state.dataSurfaceGeometry->SurfaceTmp(SurfNum).ExtBoundCond = unenteredAdjacentSpaceSurface;
+                    // check OutsideFaceEnvironment for legal zone
+                    Found = Util::FindItemInList(
+                        state.dataSurfaceGeometry->SurfaceTmp(SurfNum).ExtBoundCondName, state.dataHeatBal->space, state.dataGlobal->numSpaces);
+                    ++NeedToAddSurfaces;
+
+                    if (Found == 0) {
+                        ShowSevereError(state,
```

Comment:
Severe error is space BC is specified, but the other side space name is not found.

Reply:
Yes, reasonable.